Add tool usage reflection questions to Adagrams (#60)
Ada-Developers-Academy/core-projects: adagrams/submit.md

##### !end-answer
### !end-challenge
<!-- prettier-ignore-end -->

## Tool Usage Reflection

<!-- prettier-ignore-start -->
### !challenge
* type: paragraph
* id: 0a403185-cde2-4a5b-9bfb-7333ff3db96h
* title: Tool Usage Reflection
##### !question

> If you used AI tools while working on this project, please share links to the AI chats you used to discuss the project and answer the following questions:

Reviewer: I don't think what I'm about to suggest is **not** important to the change, but more so for organization! How would we feel about a separate submission box for the AI chat link? I don't think that would be a huge issue to keep it them together, but it would be a small nicety for us to know it would always be in the same spot. Let me know what you think!

Author: I'm good breaking that into a separate box and asking them to use a space to separate links if there are more than one 👍

Author: I'm good with that. I had an initial thought to keep them together in case someone didn't use any AI they wouldn't have multiple things to add "N/A" to, but if it might be easier for instructors, I'm happy to split those out.

- How did you confirm that the information the AI shared was up to date and accurate?

> - What feedback did you find useful or think would be helpful to keep in mind for the future?
> - What feedback was not useful or relevant?

Reviewer 2: I'm not sure why, but in these instances, "feedback" makes it seem like they ran a completed project through AI to receive feedback. I wonder if something like "suggestions" might be more appropriate.

```suggestion
- What suggestions did you find useful or think would be helpful to keep in mind for the future?
- What suggestions were not useful or relevant?
```

Author: That sounds good to move it away from the language we use for a full project review 👍

##### !end-answer
### !end-challenge
<!-- prettier-ignore-end -->

> ## Tool Usage Reflection

Author: Make title plural since there are 2 main reflection questions

```suggestion
## Tool Usage Reflections
```

<!-- prettier-ignore-start -->
### !challenge
* type: paragraph
* id: 0a403185-cde2-4a5b-9bfb-7333ff3db96h
* title: Tool Usage Reflection
##### !question

If you used AI tools while working on this project, please reflect on your usage and answer the following questions:
- How did you confirm that the information the AI shared was up to date and accurate?
- What suggestions did you find useful or think would be helpful to keep in mind for the future?
- What suggestions were not useful or relevant?

Please respond "N/A" if you did not use any AI resources.

##### !end-question
##### !placeholder

My reflections on the questions above are...

##### !end-placeholder
### !end-challenge
<!-- prettier-ignore-end -->

<!-- prettier-ignore-start -->
### !challenge

> * type: short-answer

Author: Happy to swap that over!
